### ISSUE TITLE
regression with 4.1.2 version

### ISSUE THREAD

**Issue by jean-mercier-hivebrite** [aiven/terraform-provider-aiven#1104]
## What happened?
When we plan with the new version we got this error

```
Provider schema difference:   &tfprotov6.Schema{
        Version: 0,
        Block: &tfprotov6.SchemaBlock{
                Version: 0,
                Attributes: []*tfprotov6.SchemaAttribute{
                        &{
                                Name:            "api_token",
                                Type:            s"tftypes.String",
                                NestedType:      nil,
-                               Description:     "Aiven authentication token. Can also be set with the
`AIVEN_TOKEN` environment variable.",
+                               Description:     "Aiven Authentication Token",
-                               Required:        true,
+                               Required:        false,
-                               Optional:        false,
+                               Optional:        true,
                                Computed:        false,
                                Sensitive:       true,
-                               DescriptionKind: s"MARKDOWN",
+                               DescriptionKind: s"PLAIN",
                                Deprecated:      false,
                        },
                },
                BlockTypes:      nil,
-               Description:     "The Aiven provider is used to interact with the various
services that Aiven offers. The provider needs to be configured with the
proper credentials before it can be used.",
+               Description:     "",
-               DescriptionKind: s"MARKDOWN",
+               DescriptionKind: s"PLAIN",
                Deprecated:      false,
        },
  }


╷
│ Error: Plugin error
│ 
│ The plugin returned an unexpected error from
│ plugin6.(*GRPCProvider).ValidateProviderConfig: rpc error: code = Unknown
│ desc = got different PrepareProviderConfig PreparedConfig response from
│ multiple servers, not sure which to use
```

Force version to 
```
    aiven = {
      source  = "aiven/aiven"
      version = "= 4.1.1"
    }
```

And it's works

our code looks like 
provider "aiven" {}

the API token is given by the env var AIVEN_TOKEN

**Comment by Serpentiel** [comment]
hey, @jean-mercier-hivebrite! 👋 

thanks for reporting this, I've introduced a fix now which should address your issue

**Comment by Serpentiel** [comment]
this has now been resolved in v4.1.3, sorry for the inconvenience! :)
